test/KdTreeTest.hs

```diff
@@ -59,6 +59,14 @@ prop_removePreservesInvariant points pKill =
     Kd.allSubtreesAreValid $ tree `Kd.remove` pKill
     where tree = Kd.fromList points
 
+prop_removeEdgeTest::Int->Bool
+prop_removeEdgeTest dummy = ret
+  where
+    a = Kd.Point3d 2 2 2
+    b = Kd.Point3d 2 3 3 --Change this line to Kd.Point3d 3 3 3,Kd.remove will work.
```

Comment:
I don't understand the comment. Should there be a third point `c` here, that's at (3, 3, 3)?

Reply:
b = Kd.Point3d 2 3 3